docs/blogs/reshare.html

```diff
@@ -0,0 +1,125 @@
+            <span class="horizontal-logo-symbol">*</span>
+            <span class="horizontal-logo-symbol">-</span>
+            <span class="edge-logo-symbol">+</span>
+        </div>
+    </div>
+    <div class="content">
+        <h1>Reconfiguration + Reshare with <code>consensus::threshold_simplex</code></h1>
+        <div class="meta">
+            <div class="author">By <a href="https://x.com/vex_0x">Ben Clabby</a></div>
+            <div class="date">October 15, 2025</div>
+        </div>
+        <p>An ongoing challenge with maintaining distributed validator sets is reconfiguring the validator committee as well as refreshing secret key material amongst active participants. Networks need to grow and shrink the committee without touching the brakes; Validator committees are rarely static in practice, e.g. to allow for staking/unstaking on Proof of Stake networks, or rotating the authorities on Proof of Authority networks.</p>
+        <p>Validator committee reconfiguration, in many modern blockchains, normally entails injecting out-of-band key material generated and maintained by the new validators. Most production signing schemes expect each operator to originate and safeguard their own long-lived private keys, and they provide no way for incumbents to collaboratively derive replacements. Without a distributed key generation primitive, there is no safe channel to refresh the collective secret in-band, so the only practical option is to deliver fresh key material through external procedures. <code>examples/reshare</code> shows a different path: drive a continuous BLS12-381 reshare entirely through the replicated log and require validators to rotate key material every epoch.</p>
+        <img src ="/imgs/reshare.png" alt="Reshare Flow" />
+        <p>The example combines <code>consensus::threshold_simplex</code>, <code>cryptography::bls12381::dkg</code>, and <code>p2p::authenticated</code> into a single demonstration that you can run end to end. Because every interaction in the reshare is in the log, each participant has a transcript of who proposed what, which shares were acknowledged, and how the next committee was formed.</p>
+
+        <h2>A refresher on <code>consensus::threshold_simplex</code> and <code>examples/vrf</code></h2>
+        <p>The <code>consensus::threshold_simplex</code> module utilizes BLS12-381 threshold signatures so that a group of validators can sign consensus messages. Each active validator has a share of the signing key. A threshold of these shares is required to sign a message.</p>
+        <p><a href="https://github.com/commonwarexyz/monorepo/tree/main/examples/vrf"><code>examples/vrf</code></a> showcases a continuous key reshare process to generate randomness, but with one flaw: participants trust a central "<a href="https://docs.rs/commonware-cryptography/latest/commonware_cryptography/bls12381/dkg/arbiter/struct.Arbiter.html">Arbiter</a>" to track commitments, acknowledgements, and reveals.</p>
```

Comment:
Maybe make it more explicit that your new example does not have this flaw? I noticed for example that the image you show later includes the "Arbiter" which would imply that it suffers from the same flaw?